contrib/azure/azure-user-data-template

```diff
@@ -25,3 +25,18 @@ coreos:
         What=/dev/sdc
         Where=/var/lib/docker
         Type=btrfs
+    - name: etcd.service
+      command: start
+      content: |
+        [Unit]
+        Description=etcd
+        [Service]
+        User=etcd
+        PermissionsStartOnly=true
+        Environment=ETCD_DATA_DIR=/var/lib/etcd
+        Environment=ETCD_NAME=%m
+        ExecStart=/usr/bin/etcd
+        Restart=always
+        RestartSec=10s
+        LimitNOFILE=1048576
```

Comment:
Let's change `LimitNOFILE` to `LimitNOFILE=40000` and remove `LimitNPROC` so this matches the proposed upstream change in coreos/coreos-overlay#1124.